packages/apidom-ls/src/config/common/schema/lint/missing-core-fields-openapi-3-0.ts

```diff
@@ -0,0 +1,61 @@
+import { DiagnosticSeverity } from 'vscode-languageserver-types';
+
+import ApilintCodes from '../../../codes';
+import { LinterMeta } from '../../../../apidom-language-types';
+import { OpenAPI30 } from '../../../openapi/target-specs';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
IMHO this comment is not needed here.

```suggestion
```

Reply:
this seems to be needed because of the underscore